pyinfra/facts/apt.py

```diff
@@ -1,13 +1,13 @@
 from __future__ import annotations
 
 import re
+from typing import List
```

Comment:
You don't actually need to import this, as [it's deprecated](https://docs.python.org/3/library/typing.html#typing.List). You should use `list` now as type annotation